src/section/mod.rs

```diff
@@ -95,9 +96,19 @@ impl Section {
         }
 
         // TODO: handle forks
-        self.chain
-            .merge(other.chain)
-            .map_err(|_| Error::InvalidMessage)?;
+        match self.chain.merge(other.chain.clone()) {
+            Ok(()) => (),
+            Err(_) => {
+                error!(
+                    "fork attempt detected: new chain: {:?}, new prefix: ({:b}), current chain: {:?}, current prefix: ({:b})",
```

Comment:
the error could due to something else?
say chain 1 [a,b,c] and chain 2 [d,e,f] ?

maybe change to: `potential fork detected: ...` ?

Reply:
Well, depending on how you look at it, that can also be considered a fork. Anyway, this is probably just temporary. With the new BRB membeship thing this will probably go away. I just wanted to check whether we get forks with the stress test. And we do.